src/content/pages/faq.mdx

```diff
@@ -13,12 +13,11 @@ subtitle: Frequently Asked Questions about EuroPython
   <li>**11 January 2025** [Speaker Mentorship Programme](/programme/mentorship/) sign-up opens</li>
   <li>**21 January 2025:** Speaker Mentorship signup closes</li>
   <li>**30 January 2025:** [Call for Reviewers](https://forms.gle/4GTJjwZ1nHBGetM18) & review process begin</li>
-
+  <li>**3 February 2025:** Call for Proposals (CfP) closes (<b>Extension</b>)</li>
 </ul>
 
 <ul className="milestone-todo">
   <li><strike>**27 January 2025:** Call for Proposals (CfP) closes</strike></li>
```

Comment:
```suggestion
  <li>**27 January 2025:** Call for Proposals (CfP) closes</li>
  <li>**3 February 2025:** Extension of Call for Proposals (CfP) closes</li>
</ul>

<ul className="milestone-todo">
```

Or remove the extension part completely and pretend that we only had this later date.

Reply:
I think it makes sense to keep info about the fact that the CFP was extended. See the new version.

Reply:
So you removed the initial closing date for the cfp on purpose now? - Just checking. Not objecting.

Reply:
Yeah, because the CfP didn't close on the 27th. Only on the 3rd (after an extension).